pkg/leader/leader.go

```diff
@@ -109,7 +109,33 @@ func Become(ctx context.Context, lockName string) error {
 			log.Info("Became the leader.")
 			return nil
 		case apierrors.IsAlreadyExists(err):
-			log.Info("Not the leader. Waiting.")
+			existing := &corev1.ConfigMap{}
+			key := crclient.ObjectKey{Namespace: ns, Name: lockName}
+			err := client.Get(ctx, key, existing)
```

Comment:
```suggestion
			err := client.Get(ctx, types.NamespacedName{Name: "lockName", Namespace: ns}, existing)
```

Reply:
[`crclient.ObjectKey` is a type alias of `types.NamespacedName`](https://github.com/kubernetes-sigs/controller-runtime/blob/198197fe0fd7553cadd8afcd3338f8ee4b8a7e3f/pkg/client/interfaces.go#L30) and we're using `key := crclient.ObjectKey{}` elsewhere in the file.

So for consistency reasons, I disagree with the suggestion and I think @asmacdo made the correct choice.

Reply:
Oks. But I think the key can go direct. Has no need to the extra line.

Reply:
At that point, it's a nit. Other areas of the file instantiate `key` in one line and use it in the next.